components/brave_wallet_ui/common/hooks/market-data-management.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import Fuse from 'fuse.js'
```

Comment:
`fuse.js` is used in swap but we haven't really done an audit of it, FYI. it should probably be pinned to an exact version at minimium.

Reply:
@diracdeltas Pinned to minimum version